app/src/main-process/main.ts

```diff
@@ -273,11 +277,18 @@ function createWindow() {
   mainWindow = window
 }
 
-/** Get the main window, creating it if necessary. */
-function getMainWindow(): AppWindow {
-  if (!mainWindow) {
-    createWindow()
-  }
+/**
+ * Register a function to be called once the window has been loaded. If the
+ * window has already been loaded, the function will be called immediately.
+ */
+function onDidLoad(fn: OnDidLoadFn) {
+  if (onDidLoadFns) {
+    onDidLoadFns.push(fn)
+  } else {
+    if (!mainWindow) {
+      createWindow()
+    }
 
-  return mainWindow!
+    fn(mainWindow!)
```

Comment:
Can we either a) add a comment about how createWindow always sets the mainWindow variable or b) have `createWindow` return the window instance it just created?